### PR TITLE
ZEA-2367: Add dragonfly doc

### PR DESCRIPTION
**Pull request by hackerchai**

### CODE REVIEW

File: docs/.vscode/CleanShot 2024-01-10 at 15.41.32@2x.png
Comment (MichaelYuhe) [inline docs/.vscode/CleanShot 2024-01-10 at 15.41.32@2x.png:1]: duplicate image in wrong place

**Review by MichaelYuhe — CHANGES_REQUESTED** [review]

**Review by MichaelYuhe — APPROVED** [review]

File: docs/pages/marketplace/dragonfly.en-US.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+After you deploy the Dragonfly service, Zeabur will automatically inject the environment variables needed for connection into other services for you.
+
+- `DRAGONFLY_HOST`
+- `DRAGONFLY_PORT`
+- `DRAGONFLY_PASSWORD`
+
+Sometimes we can use `DRAGONFLY_URI` instead of the above environment variables, for example:
+
+```bash copy
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.en-US.mdx:43]:
```suggestion
```bash
```

File: docs/pages/marketplace/dragonfly.ja-JP.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Dragonfly サービスをデプロイした後、Zeaburは関連接続用の環境変数を他のサービスに自動的に注入します。
+
+- `DRAGONFLY_HOST`
+- `DRAGONFLY_PORT`
+- `DRAGONFLY_PASSWORD`
+
+時折、上記の環境変数の代わりに `DRAGONFLY_URI` を使用することもあります、例えば：
+
+```bash copy
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.ja-JP.mdx:43]:
```suggestion
```bash
```

File: docs/pages/marketplace/dragonfly.zh-CN.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+当你部署 Dragonfly 服务后，Zeabur 会自动帮你注入相关连接使用的环境变量到其他的服务中。
+
+- `DRAGONFLY_HOST`
+- `DRAGONFLY_PORT`
+- `DRAGONFLY_PASSWORD`
+
+有时候我们可以使用`DRAGONFLY_URI` 来取代上面的环境变量，例如：
+
+```bash copy
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-CN.mdx:43]:
```suggestion
```bash
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+title: 部署 Dragonfly 服務
+ogImageTitle: 部署 Dragonfly 服務
+ogImageSubtitle: 在 Zeabur 部署 Dragonfly 服務
+---
+
+# Deploy Dragon Service
+
+[Dragonfly](https://www.dragonflydb.io/) 是一種簡單、高性能、低成本的內存數據存儲。Dragonfly 與 Redis API 完全兼容，但沒有 Redis 管理的複雜性。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:9]:
```suggestion
[Dragonfly](https://www.dragonflydb.io/) 是一種簡單、高效能、低成本的記憶體內資料儲存空間。Dragonfly 與 Redis API 完全相容，但沒有 Redis 管理如此複雜。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+這篇文章將教你如何通過服務市場在 Zeabur 上部署自己的 Dragonfly 服務。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:11]:
```suggestion
這篇文章將教你如何透過服務市場，在 Zeabur 上部署自己的 Dragonfly 服務。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+## 步驟一：建立項目
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:13]:
```suggestion
## 步驟一：建立專案
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+第一步，我們需要建立一個項目。進入 [項目](https://dash.zeabur.com/projects) 頁面，你會看到一個項目列表。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:15]:
```suggestion
第一步，我們需要建立一個專案。進入 [專案](https://dash.zeabur.com/projects) 頁面，你會看到一個專案列表。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+點擊列表末尾的 "創建項目" 按鈕以創建一個新項目。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:17]:
```suggestion
點選列表結尾的「建立專案」按鈕來建立一個新專案。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+![Create Project](/deploy/common/create-project.en-US.png)
+
+## 步驟二：部署 Dragonfly 服務
+
+你可以在服務市場中一鍵部署 Dragonfly 服務。
+
+按下 "cmd + k" 或者點擊 "Deploy New Service" 按鈕，然後選擇 "服務市場"。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:25]:
```suggestion
按下 "cmd + k" 或者點選 "Deploy New Service" 按鈕，然後選擇「服務市場」。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+![Select Prebuilt](/deploy/common/select-prebuilt-service.en-US.png)
+
+在列表中找到 Dragonfly 並點擊它以開始部署。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:29]:
```suggestion
在列表中找到 Dragonfly 並點選它以開始部署。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+![Select Dragonfly](/databases/dragonfly/deploy-dragonfly.en-US.png)
+
+## 步驟三：獲取連接信息
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:33]:
```suggestion
## 步驟三：取得連線資訊
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+當你部署 Dragonfly 服務後，Zeabur 會自動幫你注入相關連接使用的環境變量到其他的服務中。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:35]:
```suggestion
部署完 Dragonfly 服務後，Zeabur 會自動幫你注入和連線相關使用的環境變數到其他的服務中。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+- `DRAGONFLY_HOST`
+- `DRAGONFLY_PORT`
+- `DRAGONFLY_PASSWORD`
+
+有時候我們可以使用`DRAGONFLY_URI` 來取代上面的環境變量，例如：
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:41]:
```suggestion
可以選擇用 `DRAGONFLY_URI` 來取代上面的環境變數，例如：
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+```bash copy
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:43]:
```suggestion
```bash
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+redis://:${DRAGONFLY_PASSWORD}@${DRAGONFLY_HOST}:${DRAGONFLY_PORT}
+```
+
+我們也可以在控制台的 "Instructions" 板塊找到連接信息
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:47]:
```suggestion
我們也可以在控制台的「Instructions」區塊找到連線資訊。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+![Get Connection Info](/databases/dragonfly/connection-dragonfly.en-US.png)
+
+## 步驟四：開始使用
+
+你可以在你的應用或者客戶端中使用上面的連接信息來連接 Dragonfly 服務。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:53]:
```suggestion
你可以在你的應用程式或者用戶端中，使用上面的連線資訊來連線到 Dragonfly 服務。
```

File: docs/pages/marketplace/dragonfly.zh-TW.mdx [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+因為 Dragonfly 與 Redis API 完全兼容，所以你可以使用任何支持 Redis 的客戶端來連接 Dragonfly 服務。
```
Comment (pan93412) [inline docs/pages/marketplace/dragonfly.zh-TW.mdx:55]:
```suggestion
因為 Dragonfly 與 Redis API 完全相容，所以你可以使用任何支援 Redis 的用戶端來連接 Dragonfly 服務。
```

**Review by pan93412 — DISMISSED** [review]
1. No `copy` after `bash`.
2. Consider using Yuhang's Traditional Chinese dictionary or GPT 4 to convert zh-TW contents.

**Review by MichaelYuhe — APPROVED** [review]

**Pull request merged**